scrapy/__init__.py

```diff
@@ -35,5 +35,6 @@
 from scrapy.http import Request, FormRequest
 from scrapy.selector import Selector
 from scrapy.item import Item, Field
+from scrapy.pipelines import ItemPipeline
```

Comment:
Why did you put this here? To me it seems more logical to import this within the spiders itself. Doing this does fail 14 extra tests, but this also because the tests expect the import now.